src/main/java/me/totalfreedom/totalfreedommod/rank/Title.java

```diff
@@ -7,6 +7,7 @@
 {
 
     DEVELOPER("a", "Developer", ChatColor.DARK_PURPLE, "Dev"),
+    FOUNDER("the", "Founder", ChatColor.DARK_RED, "TF Founder"),
```

Comment:
Thats for you to figure out, as I say, it should be generic to just "Founder" or "Server Founder" not "TF Founder"